src/main/groovy/com/github/benmanes/gradle/versions/updates/DependencyUpdatesReporter.groovy

```diff
@@ -175,27 +176,29 @@ class DependencyUpdatesReporter {
     )
   }
 
+  private static void updateKey(Map<String, String> existingKey) {
+    int index = existingKey['name'].lastIndexOf('[')
+    existingKey['name'] = (index == -1) ? existingKey['name'] : existingKey['name'].substring(0, index)
+  }
+
```

Comment:
It's only two lines, but the fact that this was repeated three times bothered me, so I abstracted it.